test/template-tests/Expand-AzureRMTemplate.ps1

```diff
@@ -153,7 +153,8 @@ function Expand-AzureRMTemplate
             #*$TemplateFileName (the name of the azure template file)
             $templateFileName = $TemplateFullPath | Split-Path -Leaf
             #*$IsMainTemplate (if the TemplateFileName is named mainTemplate.json)
-            $isMainTemplate = 'mainTemplate.json', 'azureDeploy.json' -contains $templateFileName
+            $isMainTemplate = 'mainTemplate.json', 'azureDeploy.json' -contains $templateFileName -or
```

Comment:
why the "or" and not just add prereq.azuredeploy.json on line 156 in the collection?

Reply:
Originally I thought that prereqs would be wildcarded.  Feel free to just add that to the list of -contains.